sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2981,6 +3106,40 @@ public partial class RegionForOnlineOffline
         public RegionForOnlineOffline(string region) { }
         public string Region { get { throw null; } }
     }
+    public partial class RestorableGremlinDatabaseGetResult : Azure.ResourceManager.Models.ResourceData
```

Comment:
Shall we rename this class to `RestorableGremlinDatabase` if this new name does not collide with any other existing classes? If it collides, we could try something like `RestorableGremlinDatabaseData`

Reply:
We already have this change in preview version and we are just making it stable. Also, we are following similar convention for existing APIs of sql and mongo so we decided to use same for these new ones as well to remain consistent.